Sources/Afluent/SequenceOperators/TimerSequence.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  TimerSequence.swift
+//  Afluent
+//
+//  Created by Annalise Mariottini on 11/9/24.
+//
+
+import Foundation
+
+@available(macOS 13.0, iOS 16.0, watchOS 9.0, tvOS 16.0, visionOS 1.0, *)
+extension AsyncSequences {
+    /// A sequence that repeatedly emits a time duration offsetn on a given interval.
```

Comment:
Minor typo but also I don't think it emits a duration, I think it emits an instant?